exercises/concept/black-jack/black_jack.py

```diff
@@ -57,7 +58,7 @@ def can_split_pairs(card_one, card_two):
 def can_double_down(card_one, card_two):
     """Determine if a blackjack player can place a double down bet.
 
-    :param card_one, card_two: str - first and second cards in hand.
+    :param card_one, card_two: str - first and second cards dealt.
```

Comment:
```suggestion
    :param card_one, card_two: str - first and second cards in hand.
```

Reply:
Woops. Totally forgot about the exemplar. Done.

Reply:
We also need to change `dealt` to `hand` for this last task, to match the instructions.

Reply:
Should the docstrings be consistent between "dealt" and "in hand" or are they intentionally different?

Reply:
They should be consistent with what the instructions say.  I don't want there to be a debate between what we meant by "hand" and what we mean by "dealt" or "given".

Reply:
Sounds good.